README.md
# Getting Started with Create React App

<a href='./History.md'></a>

This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).

## Available Scripts

In the project directory, you can run:

### `npm start`

Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.

The page will reload when you make changes.\
You may also see any lint errors in the console.

### `npm test`

Launches the test runner in the interactive watch mode.\
See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.

### `npm run build`

Builds the app for production to the `build` folder.\
It correctly bundles React in production mode and optimizes the build for the best performance.

The build is minified and the filenames include the hashes.\
Your app is ready to be deployed!

See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.

### `npm run eject`

**Note: this is a one-way operation. Once you `eject`, you can't go back!**

If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.

Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.

You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.

## Learn More

You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).

To learn React, check out the [React documentation](https://reactjs.org/).

### Code Splitting

This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)

### Analyzing the Bundle Size

This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)

### Making a Progressive Web App

This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)

### Advanced Configuration

This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)

### Deployment

This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)

### `npm run build` fails to minify

This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
<iframe width="560" height="315" src="https://www.youtube.com/embed/ieSgPWjnMZ8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
Porfirio Díaz
Nació en la ciudad de Oaxaca un 15 de septiembre de 1830, ¡sí, el mismo día de la Independencia de México! Pero
Porfirio no quería ser un simple espectador de la historia, así que ingresó al Instituto de Ciencias y Artes de Oaxaca para estudiar leyes, ¡quería ser un abogado muy top!

Pero, ¡oh no! Los conflictos políticos del país no permitieron que Porfirio concluyera sus estudios, así que decidió luchar contra los conservadores en la Guerra de Reforma. Participó en la Batalla de Puebla durante la intervención francesa y logró huir cuando las fuerzas franco-mexicanas tomaron su ciudad natal.

Pero no se rindió, ¡al contrario! Inició una serie de victorias militares y en 1876 triunfa la Revolución de Tuxtepec. ¡Y tachán! El 5 de mayo de 1877 toma posesión de la presidencia de México. Pero su reinado no terminaría ahí, ¡no señor! En 1884 asumió la presidencia nuevamente y no la abandonó hasta 1911, ¡ni más ni menos que 27 años en el poder! Pero entonces llegó Francisco I. Madero y convocó a la rebelión.

Porfirio Díaz renunció a la presidencia en mayo de 1911 y abandonó el país. Y lamentablemente, amigos, ¡falleció en París, Francia, el 2 de julio de 1915! Aunque sus ideas y acciones siguen siendo objeto de controversia, su figura es parte importante de la historia de México y su legado sigue siendo recordado en la actualidad.


History.md
<html>
  <body>
    <header className='App-header'>
        <h1><strong>Historia de México</strong></h1>
        <navigator><ul><li className='navigator'><a href='https://www.gob.mx'>México</a></li><li className='navigator'><a href='https://www.curistoria.com/'> Internacional</a></li></ul></navigator>
      </header>
      <main>
      <h1>Personajes Importantes de México</h1>
      <ul>
        <li>Benito Juarez(1806-1872) <span className='cita'>"Entre los individuos, como entre las naciones, el respeto al derecho ajeno es la paz"</span></li>
          <a href='https://youtu.be/79rBOsrcX6I'>Ver video en Youtube</a>
        <li className='Porfirio'>Porfirio Díaz(1830-1915)<span className='cita'>"Orden y Progreso"</span></li>
        <p>Porfirio Díaz Empezó su carrera militar en 1854, al unirse al plan de Ayutla Debido a la persecusión que sufría por parte de Santa Anna, todo esto en un papel menor.
          Posteriormente, la Guerra de Reforma de 1857 a 1860, fue Gobernador y Comandante Militar del Istmo en Oaxaca y logró alcanzar el grado de Coronel.
          Además de participar en la mitica batalla del 5 de mayo, por ese y otros motivos, el presidente Juarez le otorgó el grado de General de División el 10 de octubre de 1863. en reconocimiento a sus actos.
          En noviembre de 1871, con la reelección del Presidente Juárez, se levantó en armas y proclamó el Plan de la Noria, sin lograr el éxito esperado, su movimiento fue derrotado. Posterior a ello se rebeló contra lerdo (Plan de Tuxtepec) con el cual se hizo del poder, para después forjar un estricto regimén que, a pesar de que modernizó el país, reprimió toda oposición, además de permitir que se maltratara a los mexicanos más pobres en las Haciendas minas y fabricas, algo que llegaria a su limte con las huelgas de Cananea(1909) y Rio Blanco (1907), por lo que Diaz se vió obligado a dimitir del cargo en 1911
        </p>
        <li className='Li'>Miguel Hidalgo(1753-1811)<span className='cita'>"¡Viva la religión!, ¡viva nuestra madre santísima de Guadalupe!, ¡viva Fernando VII!, ¡viva la América y muera el mal Gobierno!"</span></li>
      </ul>
      </main>
      <footer>
      <h3>Fuentes</h3>
      <a className='PD' href='https://www.gob.mx/sedena/documentos/2-de-julio-de-1915-aniversario-luctuoso-del-general-porfirio-diaz'>Nacional, S. D. L. D. (s. f.). 2 de julio de 1915, aniversario luctuoso del General Porfirio Diaz. gob.mx. https://www.gob.mx/sedena/documentos/2-de-julio-de-1915-aniversario-luctuoso-del-general-porfirio-diaz</a>
      </footer>
  </body>
</html>


src/App.js
import './App.css';

function App() {
  return (
    <div className='App'>
      <header className='App-header'>
        <h1><strong>Historia de México</strong></h1>
        <navigator><ul><li className='navigator'><a href='https://www.gob.mx'>México</a></li><li className='navigator'><a href='https://www.curistoria.com/'> Internacional</a></li></ul></navigator>
      </header>
      <main>
       <ul>
        <li><strong>Benito Juarez</strong><span className='cita'> Entre los individuos como entre las naciones, el respeto al derecho ajeno es la paz</span></li>
        <iframe width="560" height="315" src="https://www.youtube.com/embed/79rBOsrcX6I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
        <li><strong>Miguel Hidalgo</strong><span className='cita'>“¡Viva la independencia! ¡Viva la América! ¡Muera el mal gobierno!”
- <a href='https://akifrases.com/autor/miguel-hidalgo-y-costilla'>https://akifrases.com/autor/miguel-hidalgo-y-costilla</a>
</span></li>
<iframe width="560" height="315" src="https://www.youtube.com/embed/ieSgPWjnMZ8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
        <li><strong>Porfirio Diaz</strong><span className='cita'> Orden y porgreso</span></li>
        <p>Nació en la ciudad de Oaxaca un 15 de septiembre de 1830, ¡sí, el mismo día de la Independencia de México! Pero Porfirio no quería ser un simple espectador de la historia, así que ingresó al Instituto de Ciencias y Artes de Oaxaca para estudiar leyes, ¡quería ser un abogado muy top!

Pero, ¡oh no! Los conflictos políticos del país no permitieron que Porfirio concluyera sus estudios, así que decidió luchar contra los conservadores en la Guerra de Reforma. Participó en la Batalla de Puebla durante la intervención francesa y logró huir cuando las fuerzas franco-mexicanas tomaron su ciudad natal.

Pero no se rindió, ¡al contrario! Inició una serie de victorias militares y en 1876 triunfa la Revolución de Tuxtepec. ¡Y tachán! El 5 de mayo de 1877 toma posesión de la presidencia de México. Pero su reinado no terminaría ahí, ¡no señor! En 1884 asumió la presidencia nuevamente y no la abandonó hasta 1911, ¡ni más ni menos que 27 años en el poder! Pero entonces llegó Francisco I. Madero y convocó a la rebelión.

Porfirio Díaz renunció a la presidencia en mayo de 1911 y abandonó el país. Y lamentablemente, amigos, ¡falleció en París, Francia, el 2 de julio de 1915! Aunque sus ideas y acciones siguen siendo objeto de controversia, su figura es parte importante de la historia de México y su legado sigue siendo recordado en la actualidad.</p>
       </ul>
      </main>
      <footer>
      <h3>Fuentes</h3>
      <a className='PD' href='https://www.gob.mx/sedena/documentos/2-de-julio-de-1915-aniversario-luctuoso-del-general-porfirio-diaz'>Nacional, S. D. L. D. (s. f.). 2 de julio de 1915, aniversario luctuoso del General Porfirio Diaz. gob.mx. https://www.gob.mx/sedena/documentos/2-de-julio-de-1915-aniversario-luctuoso-del-general-porfirio-diaz</a>
      </footer>
    </div>
  );
}

export default App;
